tracing-subscriber/src/fmt/mod.rs

```diff
@@ -237,12 +237,12 @@ pub struct CollectorBuilder<
 /// })
 /// ```
 ///
-/// [`SubscriberBuilder`]: struct.SubscriberBuilder.html
+/// [`CollectorBuilder`]: struct.CollectorBuilder.html
 /// [formatting subscriber]: struct.Collector.html
```

Comment:
reference to subscriber with a small s, probably needs to be changed...

Reply:
yup, fixed!